circle.yml

```diff
@@ -0,0 +1,66 @@
+version: 2
+
+jobs:
+  macos:
+    macos:
+      xcode: "10.1.0"
+    steps:
+      - checkout
+      - run: swift build
+      - run: swift test
+
+  linux:
+    docker:
+      - image: swift:4.2
+    steps:
+      - checkout
+      - run: 
+          name: Compile code
+          command: swift build
+      - run: 
+          name: Run unit tests
+          command: swift test
+
+  linux-41:
+    docker:
+      - image: codevapor/swift:4.1
+    steps:
+      - checkout
+      - run: 
+          name: Compile code
+          command: swift build
+      - run: 
+          name: Run unit tests
+          command: swift test
+
+  linux-release:
+    docker:
+      - image: swift:4.2
+    steps:
+      - checkout
+      - run: 
+          name: Compile code with optimizations
+          command: swift build -c release
+
+
+workflows:
+  version: 2
+  tests:
+    jobs:
+      - linux
+      - linux-41
+      - linux-release
+       - macos
+
+  nightly:
```

Comment:
I don't think we need nightly really going forward. Haven't gotten much use out of them.